### PR TITLE
Refactored to pull data based on product id, as well as changed the DB seeding script. Updated readme

### PR DESCRIPTION
**Pull request by michaelrtobias**

### CODE REVIEW

**Comment by michaelrtobias** [comment]
@twzkraus Thank You!

**Comment by michaelrtobias** [comment]
circle CI is going to fail. Will fix that at some point

File: DB/seed.js [hunk 1]
```diff
@@ -92,9 +98,10 @@ var fakeImageDataCreator = function(numberOfEnries) {
   return dataEntries;
 }
 
-var fakeData = fakeImageDataCreator(600);
+var fakeData = fakeImageDataCreator(700);
 var data = realData.concat(fakeData)
 
+console.log(fakeData)
```
Comment (twzkraus) [inline DB/seed.js:104]: you left a console.log in here. Pretty minor.

**Review by twzkraus — COMMENTED** [review]
Looks pretty good!

**Pull request merged**